### ISSUE TITLE
Save personal information

### ISSUE THREAD

**Issue by TamaraHoock** [amosproj/amos2022ss04-digital-identity#8]
## User story
1. As a user (HR employee)
2. I want / need to save the added personal information of the employee
3. So that it is stored in the database.

## Acceptance criteria
*

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by jackDS008** [comment]
duplicate with #7
